docs/sdk/serverless-signing/combining-decryption-shares.md

```diff
@@ -0,0 +1,71 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Decrypting and Combining Within an Action
+
+## Overview
+Within a Lit action, you may choose to combine a `decryption shares` from within the context of an action. Meaning signature shares from each node will be combined and given to every node which recieved a request to execute the given action. Combining within an action may be useful if you wish to take advantage of the `Secure Compute Enviorment` offered by the Lit network. Actions which take combine signatures from within their own execution context will no provide the shares of the signatures to the client. So all information will stay in the trusted execution enviorment (TEE).
```

Comment:
Copy suggestion:

Current: 
"Within a Lit action, you may choose to combine a `decryption shares` from within the context of an action. Meaning signature shares from each node will be combined and given to every node which recieved a request to execute the given action. Combining within an action may be useful if you wish to take advantage of the `Secure Compute Enviorment` offered by the Lit network. Actions which take combine signatures from within their own execution context will no provide the shares of the signatures to the client. So all information will stay in the trusted execution enviorment (TEE)."

Suggestion: 
"[Decryption with Lit](https://developer.litprotocol.com/v3/sdk/access-control/quick-start) is typically done client-side by an authorized party at the time of access. The `decryptAndCombine` function allows you to decrypt data within a Lit Action. This function is useful for performing operations over sensitive data, where the data itself remains private within the confines of each [Lit node's TEE](https://developer.litprotocol.com/v3/resources/how-it-works#1-lit-nodes). 

When you call `decryptAndCombine`, the `decryption shares` from each node are collected and aggregated on a single Lit node before they are combined. "

^^ that function is only run on a single node right?

Reply:
@a1ttech - currently the decryption shares are provided to every node, and each recombines them.   This means that ALL nodes have a copy of the recombined data, rather than just one node.